regtest/basic/rt-drmsd/config

```diff
@@ -1,3 +1,4 @@
+plumed_modules=adjmat
```

Comment:
I will try to explain this here but it would be good to have a chat about how to proceed here.  My preference would be to move some files around a little bit and then submit another PR.

So a plumed input file that could be created for doing DRMSD (remember DRMSD is a shortcut) is as follows (it is not done this way.  Although the way described below is probably better.  That is another change that should perhaps be made).

````
# Create a vector to hold the reference distances
dref: CONSTANT VALUES=1.1,1,3,1.4....
# Calculate all the distances
d: DISTANCES ATOMS1=1,2 ATOMS2=3,4 ...
# Calculate the displacement vector
dis: DISPLACEMENT ARG=d,dref
# Transpose the displacement vector
disT: TRANSPOSE ARG=dis
# Matrix multiply the transpose by the displacement to get the distance
d: MATRIX_PRODUCT_DIAGONAL ARG=disT,dis
````

This uses functionality for dealing with matrices.  All of this functionality is currently in adjmat as I wrote all the matrix stuff initially with adjacency matrices in mind.  I then started using it in other places because matrix multiplication is really useful!  So I should probably have a rethink of how the files are organised into modules.  That is quite a big job though that should maybe be done in a different PR.

Reply:
One issue that I see with the shortcuts is the following.

In a recent commit e912958d594be842d468a1480eec70478d22e0f1 I used a short script to remove unneeded dependencies. The script looks like this:
```
for file in */Makefile ; do

if grep -q "USE=" $file ; then
  deps=$(grep USE= $file | awk '{for(i=2;i<=NF;i++) print($i "/")}')
  for dep in $deps ; do
    git grep -q "^ *# *include \"$dep" ${file%Makefile}  || echo ERROR $file $dep
  done
fi

done
```

It checks if a module was USEd by mistake. And the fact that the build with all modules enables (on GitHub Actions) confirms that all needed modules where USEd.

However, we now have other dependencies that are a bit more difficult to track. For instance, the code might build with a set of modules, some action would seem available, but then would lead to an error at runtime because they generate a shortcut that need another module.

I don't have a solution however...